docs/dependencies-windows.md

```diff
@@ -129,7 +129,7 @@ The following steps require you to store dynamic libraries in the `pupil_externa
 
 #### FFMPEG
 
-- Download FFMPEG v4.0 Windows shared binaries from [ffmpeg](http://ffmpeg.zeranoe.com/builds/)
+- Download FFMPEG v4.0 Windows shared binaries from [ffmpeg](https://github.com/BtbN/FFmpeg-Builds/releases)
 - Unzip ffmpeg-shared to your work dir
```

Comment:
```suggestion
- [Download FFMPEG v4.0 Windows **shared** binaries](https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2020-10-12-12-31/ffmpeg-n4.3.1-20-g8a2acdc6da-win64-lgpl-shared-4.3.zip)
- Unzip `ffmpeg-*-shared-4.3.zip` to your work dir
```
Given that https://github.com/BtbN/FFmpeg-Builds/releases is offering a bunch of different versions, it might be easier to just link a fixed version here to avoid confusion.